src/stdlib_stats_distribution.fypp

```diff
@@ -0,0 +1,226 @@
+#:include "common.fypp"
+module stdlib_stats_distribution
+    !! Stdlib_stats_distribution
+    !! This module contains a large number of probability distribution and statistical functions
+    !!
+    !! Continuouse Distributions
+    !!     Uniform distribution
+    !!     Normal distribution
+    !!
+    !! Discrete Distribution
+    !!     Binormial distribution
+    !!
+    use stdlib_error, only : error_stop
+    use stdlib_kinds
+    use stdlib_stats_distribution_rvs
+    implicit none
+    private
+    #:for k1, t1 in REAL_KINDS_TYPES
+    ${t1}$, parameter :: sqrt_2_pi_${t1[0]}$${k1}$ = sqrt(2.0_${k1}$ * acos(-1.0_${k1}$))
+    ${t1}$, parameter :: sqrt_2_${t1[0]}$${k1}$ = sqrt(2.0_${k1}$)
+    #:endfor
+
+    public :: random_seed
+    public :: uniform_distribution_rvs, uniform_distribution_pdf, uniform_distribution_cdf
+    public :: normal_distribution_rvs, normal_distribution_pdf, normal_distribution_cdf
+    public :: binomial_distribution_rvs, binomial_distribution_pmf, binomial_distribution_cdf
+!---------------------------------------------------------------------------------------------
+!                   Random seed
+!---------------------------------------------------------------------------------------------
+    interface random_seed
+    !! version: experimental
+    !!
+    !! Set or get seed value of probability distributions for psudorandom bumber generator 
+    !! ([Specification](../page/specs/stdlib_stats_distribution.html#description))
+    !! 
+        procedure random_distribution_seed
+    end interface random_seed
+!---------------------------------------------------------------------------------------------
+!                   Uniform Distribution
+!---------------------------------------------------------------------------------------------
```

Comment:
```suggestion

```

Reply:
as per @milancurcic , these separators add no value and is not in style, will remove in the next round.